stable/phpmyadmin/README.md

```diff
@@ -0,0 +1,80 @@
+
+## TL;DR;
+
+```console
+$ helm install stable/phpmyadmin
+```
+
+## Introduction
+
+This chart bootstraps a [Phpmyadmin](https://github.com/bitnami/bitnami-docker-phpmyadmin) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/phpmyadmin
+```
+
+The command deploys Phpmyadmin on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Phpmyadmin chart and their default values.
+
+|              Parameter               |               Description                |                         Default                         |
+|--------------------------------------|------------------------------------------|---------------------------------------------------------|
+| `image.registry`                     | PhpMyAdmin image registry                 | `docker.io`                                             |
+| `image.repository`                   | PhpMyAdmin Image name                     | `bitnami/phpmyadmin`                                     |
+| `image.tag`                          | PhpMyAdmin Image tag                      | `{VERSION}`                                             |
+| `image.pullPolicy`                   | Image pull policy                        | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
+| `image.pullSecrets`                  | Specify image pull secrets               | `nil`                                                   |
+| `phpmyadmin.serviceType`            | type of service for PhpMyAdmin frontend             | `ClusterIP`                                                  |
+| `phpmyadmin.servicePort`        | port to expose service                   | `80`                                                   |
+| `phpmyadmin.dbPort`            | database port to use to connect                  | `3360`                                     |
+| `phpmyadmin.dbSuffix`                | Database suffix if included in the same release                  | `nil`                                          |
+| `phpmyadmin.dbHost`            | database host to connect to               | `nil`          |
```

Comment:
Are you sure we should be using a top-level `phpmyadmin` key for these params? Typically we use a structure such as the following in the values.yaml:

```
service:
  type: ClusterIP
  port: 80

db:
  host: 
  port: 3360
  suffix: 
```

Reply:
I think because it is likely that this chart will be used as a dependency, it think your comment makes sense. I will change it